scripts/installer.sh

```diff
@@ -24,6 +24,7 @@ main() {
 	VERSION=""
 	PACKAGETYPE=""
 	APT_KEY_TYPE="" # Only for apt-based distros
+	APT_SYSTEMCTL_START=true # Only for apt-based distros
```

Comment:
This should be false by default. Only Kali (so far) needs the quirky behavior.

Reply:
As of today, it won't install on debian or Ubuntu without that flag set

Reply:
I'm saying, change it from "true" to "false".
Debian and Ubuntu don't need us to call systemctl after install, only Kali does.
The bug right now is that the variable is completely undefined, and we set -x at the top of the script.